tests/defaults_test.go

```diff
@@ -50,23 +52,48 @@ func TestNamespace(t *testing.T) {
 	}
 	for _, namespace := range namespaces {
 		namespace := namespace
+
+		// Create a release that installs into the given namespace and
+		// with the given name
 		opts := opts
+		opts.Name = namespace + "-release"
 		opts.Namespace = namespace
+
 		t.Run(namespace, func(t *testing.T) {
 			t.Parallel()
 
 			// Render the chart with default values
 			objs, err := chart.Render(nil, &opts, nil)
 			require.NoError(t, err, "chart render failed")
 
-			// Verify that all objects are using the supplied namespace
 			for _, obj := range objs {
 				metaObject, err := meta.Accessor(obj)
-				require.NoError(t, err, "failed to get object metadata")
+				require.NoErrorf(t, err, "failed to get object metadata for object %q with name %q",
+					obj.GetObjectKind().GroupVersionKind(), metaObject.GetName())
 
+				// Verify that all objects are using the supplied namespace
 				actualNamespace := metaObject.GetNamespace()
-				require.Equal(t, namespace, actualNamespace,
-					"deployed namespace does not match target")
+				require.Equalf(t, namespace, actualNamespace,
```

Comment:
This check ensures that all created objects have the expected labels, or else emits an error so we can identify the object missing stuff. I'm not sure if there's a more concise way to note the debug info (object kind and name), which is needed to uniquely identify a given resource